components/threading/thread_viewer/thread_viewer.scss

```diff
@@ -49,7 +49,6 @@
 
         .icon {
             margin-right: 3.4px;
-            color: rgba(63, 67, 80, 0.56);
```

Comment:
@thinkGeist please use css variable instead `rgba(var(--center-channel-text-rgb), 0.56)` 
https://github.com/mattermost/mattermost-webapp/blob/b5e57eb863b71a41f0b9b2ba0b8e517b27486dc2/sass/base/_css_variables.scss#L64

Reply:
Hey @AshishDhama this doesn't seem to have quite the intended effect. When a dark theme is applied the icon is still a dark color also, making it difficult to see:

![image](https://user-images.githubusercontent.com/9254315/173410225-48c0e7ad-3d30-4191-9d15-c0ded8de7bae.png)

Would `--center-channel-color-rgb` potentially be correct here? It seems to correspond to the "Center channel text" color in the theme options here:
![image](https://user-images.githubusercontent.com/9254315/173410462-2d124f08-7557-4f85-b063-d8c758f1ff15.png)

Here's an example using `--center-channel-color-rgb` with the "Onyx" theme applied:
![image](https://user-images.githubusercontent.com/9254315/173410628-49f3144c-b76c-4853-8020-19e7daa243fc.png)

Thanks again for the feedback!

Reply:
yes @thinkGeist ... `--center-channel-color-rgb` is the correct variable to use